src/components/NavigationBar.tsx

```diff
@@ -0,0 +1,117 @@
+/* This example requires Tailwind CSS v2.0+ */
+import { Fragment } from 'react'
+import { Disclosure } from '@headlessui/react'
+import { MdOutlineComputer, MdOutlineMenu, MdClose } from "react-icons/md";
+
+export default function NavigationBar() {
+  return (
+    <Disclosure as="nav" className="bg-dark shadow text-white">
+      {({ open }:any) => (
```

Comment:
Creio que tenha como resolver esse open sem o any

Reply:
Sure, I made the correction :)